### ISSUE TITLE
Invalid GeoJson Returned

### ISSUE THREAD

**Issue by brianherbert** [ushahidi/Ushahidi_Web#1432]
https://DEPLOYMENT_URL/json/index returns invalid GeoJson. Coordinates should be floats, not strings.

**Comment by xmasclaux** [comment]
Yes and geojson validator says : "list item is not of type number". Have you found a way to solve this bug ?
